cli/setup.go

```diff
@@ -922,7 +922,7 @@ func (opts *setupOptionsType) maybeAddHostLookup() {
 	}
 
 	// Seek to last character
-	_, err = f.Seek(-1, os.SEEK_END)
+	_, err = f.Seek(-1, io.SeekEnd)
```

Comment:
and were did that come from? we also want to cp it?

Reply:
PIpeline fails without this change because newer golang deprecafted io.SEEK_END